@here/harp-map-controls/lib/CameraKeyTrackAnimation.ts

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { MapView, MapViewEventNames, MAX_TILT_ANGLE } from "@here/harp-mapview";
+import { LoggerManager } from "@here/harp-utils";
+import THREE = require("three");
+
+import { CameraAnimation } from "./MapAnimations";
+import { CameraPose, CameraPoseOptions } from "./Utils";
+
+const logger = LoggerManager.instance.create("CameraKeyTrackAnimation");
+
+export interface ControlPointOptions extends CameraPoseOptions {
+    timestamp: number;
```

Comment:
what does the `timestamp` number mean / which units is it in?

Reply:
done